### ISSUE TITLE
[Question]The task instance does not exist

### ISSUE THREAD

**Issue by peizhuang-ws** [apache/incubator-dolphinscheduler#4657]
The dolphinscheduler version 1.3.2
Problems encountered：
After a task is executed for a period of time, a task instance does not exist
Investigation：
Check the log. There is no error information in the API log during the time period when the error is found. There is an error of "read file error" in the logger log,
[pool-2-thread-3] error org.apache.dolphinscheduler . server.log.LoggerRequestProcessor - read file error
java.nio.file .NoSuchFileException: /opt/apps/dolphinscheduler/logs/13304/42892/256494.log
at sun.nio.fs . UnixException.translateToIOException ( UnixException.java:86 )
at sun.nio.fs . UnixException.rethrowAsIOException ( UnixException.java:102 )
at sun.nio.fs . UnixException.rethrowAsIOException ( UnixException.java:107 )
at sun.nio.fs .Un ixFileSystemProvider.newByteChannel (Un ixFileSystemProvider.java:214 )
Basically once a week or two
What can we do to solve this problem

**Comment by dailidong** [comment]
you should check the worker server runtime log
if it is a cron task, you also can check the master server runtime log whether there is exception or warn messages

**Comment by peizhuang-ws** [comment]
> 您应该检查辅助服务器运行时日志
> （如果是 cron 任务），还可以检查主服务器运行时日志是否有异常或警告消息

We encountered this problem again. After checking the worker log and master log, we found no reason. Because there was an error in the log before, we couldn't find the log. We doubt whether it was caused by deleting the workflow instance. There is no way to verify it. However, because of deleting the workflow instance, the whole dolphin scheduler can't run, which is also inappropriate

**Comment by chengshiwen** [comment]
@peizhuang-ws @CalvinKirs @dailidong The dev and new 1.3.5-release has fixed the `NoSuchFileException`. See #4271 #4272 #4599
As for why the log is lost, please check the log directory and check whether it is put in `/tmp`, whether to delete regularly by `cron` @peizhuang-ws

**Comment by chengshiwen** [comment]
Close this issue since no updates for long time
